src/plugin.scss

```diff
@@ -20,6 +20,9 @@
   background:rgba(0, 0, 0, .5);
   -ms-filter: "progid:DXImageTransform.Microsoft.gradient(startColorstr=#7F000000,endColorstr=#7F000000)"; /* IE8 */
```

Comment:
I feel like we ought to get rid of these and just give IE8 a solid black background. We'll be dropping support anyway; might as well.